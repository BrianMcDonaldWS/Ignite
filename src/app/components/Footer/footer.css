.footer {
  padding: 30px;
  width: 100%;

  .copyright {
    margin: 0;
    padding-top: 30px;
    font-size: 14px;
    color: #fff;
    border-top: 1px solid #9e9e9e;
  }
}
