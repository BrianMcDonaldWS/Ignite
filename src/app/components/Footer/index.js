import React from 'react';
import makeClass from 'classnames';

import styles from './footer.css';

const Footer = () => (
  <footer className={makeClass(styles.footer, 'footer')}>
    <div className="container">
      <div className={makeClass('has-text-centered', 'content')}>
        <p className={styles.copyright}>
          Made with ğŸŒ³ğŸ”¥ğŸ’¨ by Intuit. Licensed under the{' '}
          <a href="http://opensource.org/licenses/mit-license.php">MIT</a>{' '}
          License.
        </p>
      </div>
    </div>
  </footer>
);

export default Footer;
