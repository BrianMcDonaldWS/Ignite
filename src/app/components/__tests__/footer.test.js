import React from 'react';

import Footer from '../Footer';
import renderToJson from './util';

test('Footer', () => {
  expect(renderToJson(<Footer />)).toMatchSnapshot();
});
