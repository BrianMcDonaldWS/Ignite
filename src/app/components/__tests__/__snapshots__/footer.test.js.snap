// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <div
      className="has-text-centered content"
    >
      <p
        className={undefined}
      >
        Made with ðŸŒ³ðŸ”¥ðŸ’¨ by Intuit. Licensed under the
         
        <a
          href="http://opensource.org/licenses/mit-license.php"
        >
          MIT
        </a>
         
        License.
      </p>
    </div>
  </div>
</footer>
`;
